lib/elixir/src/elixir_dispatch.erl

```diff
@@ -144,6 +144,7 @@ do_expand_import(Meta, {Name, Arity} = Tuple, Args, Module, E, Result) ->
       check_deprecation(Meta, Receiver, Name, Arity, E),
       elixir_lexical:record_import({Receiver, Name, Arity}, ?m(E, lexical_tracker)),
       elixir_locals:record_import(Tuple, Receiver, Module, ?m(E, function)),
+      elixir_lexical:record_remote(Receiver, Name, Arity, nil, ?line(Meta), ?m(E, lexical_tracker)),
```

Comment:
Preferably we would add `record_import/6` and call the lexical tracker just once and it would write to both places.